attr_pdr.go

```diff
@@ -25,14 +25,17 @@ type PDR struct {
 	Precedence      *uint32
 	PDI             *PDI
 	OuterHdrRemoval *uint8
-	FARID           *uint32
-	QERID           *uint32
-	URRID           *uint32
+	FARID           []*uint32
+	QERID           []*uint32
+	URRID           []*uint32
 	SEID            *uint64
 }
 
 func DecodePDR(b []byte) (*PDR, error) {
 	pdr := new(PDR)
+	pdr.FARID = make([]*uint32, 0)
```

Comment:
No need to initialize a slice that its length is zero because this field is already initialized by new().